src/events/index.js

```diff
@@ -160,6 +160,74 @@ class Events {
       .then((event) => toCamelCase(event));
   }
 
+  /**
+   * Gets all event types for a client
+   *
+   * API Endpoint: '/clients/:clientId/types'
+   * Method: GET
+   *
+   * @param {string} clientID The ID of the client
+   *
+   * @returns {Promise}
+   * @fulfill {Event} Information about an event
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.events
+   *   .getEventTypesByClient('CW4B1Ih6M1nNwwxk0XOKI21MVH04pGUL')
+   *   .then((event) => console.log(event))
+   *   .catch((err) => console.log(err));
+   */
+  getEventTypesByClient(clientId) {
+    if (!clientId) {
+      return Promise.reject(
+        new Error('A client ID is required for getting types')
+      );
+    }
+    return this._request.get(`${this._baseUrl}/clients/${clientId}/types`);
+  }
+  /**
+   * Gets all events by type
+   *
+   * API Endpoint: '/types/:typeId/events?facility_id=:facility_id&include[]=latest'
+   * Method: GET
+   *
+   * @param {string} id The ID of the type
```

Comment:
Personal preference here but since there are multiple id's here I think it would be more clear if this was changed to `eventTypeId` or something similar